src/ecosystem/corals/growth.jl

```diff
@@ -316,19 +522,20 @@ end
 """
     settler_cover(fec_scope, sf, TP_data, leftover_space, α, β, basal_area_per_settler)
 
+Determine area settled by recruited larvae (in m²).
+
 # Arguments
 - `fec_scope` : fecundity scope
-- `sf` : stressed fecundity
 - `TP_data` : Transition probability
-- `leftover_space` : difference between sites' maximum carrying capacity and current coral cover (k - C_s)
+- `leftover_space` : difference between sites' maximum carrying capacity and current coral cover (\$k - C_s\$)
```

Comment:
Maybe specify units of leftover_space

Reply:
The docstring mentions m^2, is that clear enough or would you like it more explicit?

Reply:
Yeah maybe just add 'Area inputs in m^2` or something, just because we've had a few issues with units mismatches it probably can't hurt. Up to you though.